<launch>
  <node name="background_store_back" pkg="bmw_percep" type="background_store" output="screen">
    <param name="bg_frames" value="10" />
    <param name="min_file" value="$(find bmw_percep)/data/kinect_back/min_bg.pcd" />
    <param name="max_file" value="$(find bmw_percep)/data/kinect_back/max_bg.pcd" />
    <remap from="/pc_in" to="/kinect_back/depth_registered/points" />
  </node>
</launch>
